### DISCUSSION TITLE
How to make popular feed as default

### DISCUSSION THREAD

**Discussion by GopalanHarish** [dailydotdev/daily#786] [Feature Requests]
Hi,

Currently, "My feed" is the default home page. 
Previously, it was "Popular" feed.

How can I set Popular feed as the default home page/initial screen?

Thank you

**Comment by idoshamun** [comment]
Hi @GopalanHarish,

This is not possible at the moment.
Once you set up your feed and define the topics you would like to follow, it becomes your new default feed.
May I know the reason for switching to popular?

**Reply by GopalanHarish** [reply]
Well, popular feed is more relevant for me. Hope I can get it back with future versions :)

Also, there is a problem with "My feed", articles' tags are sometimes incorrect.
Example: For tag "**rust**", could mean different things when we are talking about programming language (**rust** programming language) and 3D design (design metal **rust** effect in mesh).
Maybe tag for programming needs to change. I don't know.

**Reply by idoshamun** [reply]
Oh that's a good point about rust! Thanks :)
Try to tweak your feed settings maybe it will help

**Reply by GopalanHarish** [reply]
Cant you add an option in the manage > customize for coming iterations?
I don't want a custom-tailored feed. I just want to see articles that are popular when I open a new tab.
I am sure many users will find it useful as it's a basic option in many interfaces.

**Reply by idoshamun** [reply]
I understand. Let me forward it internally :)

**Reply by GopalanHarish** [reply]
thank you
